source/agora/consensus/EnrollmentManager.d

```diff
@@ -161,7 +160,7 @@ public class EnrollmentManager
 
         // create the table for enrollment data for a node itself
         this.db.execute("CREATE TABLE IF NOT EXISTS node_enroll_data " ~
-            "(key CHAR(128) PRIMARY KEY, val BLOB NOT NULL)");
+            "(key CHAR(128) PRIMARY KEY, val CHAR(66) NOT NULL)");
```

Comment:
```suggestion
            "(key CHAR(128) PRIMARY KEY, val TEXT(66) NOT NULL)");
```

Reply:
Actually, according to SQLite manual, it just ignore the length, so I'm going to make it `TEXT`